modules/ocf_www/files/lets-encrypt-update

```diff
@@ -110,136 +81,6 @@ def eligible_domains(target):
     return eligible_domains
 
 
-def expiration_date(cert_path):
-    output = subprocess.check_output((
-        'openssl', 'x509', '-enddate', '-noout',
-        '-in', str(cert_path),
-    ))
-    m = re.match(b'notAfter=(.+)$', output)
-    assert m, output
-    return dateutil.parser.parse(m.group(1).decode('ascii'))
-
-
-@contextmanager
-def make_csr(domain):
-    _, csr_path = tempfile.mkstemp()
-    try:
-        subprocess.check_call((
-            'openssl', 'req', '-new', '-sha256',
-            '-key', str(LE_VHOST_KEY),
-            '-subj', '/CN={}'.format(domain),
-            '-out', csr_path,
-        ))
-        yield csr_path
-    finally:
-        os.remove(csr_path)
-
-
-def acme_tiny(domain):
-    with make_csr(domain) as csr_path:
-        # TODO: use subprocess.run when we get python3.5
-        proc = subprocess.Popen(
-            (
-                'acme-tiny',
-                '--account-key', str(LE_ACCOUNT_KEY),
-                '--csr', csr_path,
-                '--acme-dir', '/srv/well-known/acme-challenge',
-                # '--ca', 'https://acme-staging.api.letsencrypt.org',
-            ),
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-        )
-        stdout, stderr = proc.communicate()
-    return stdout, stderr, proc.returncode
-
-
-def write_cert(cert, cert_path):
-    """Write certificate file for a domain."""
-    _, temp_cert_path = tempfile.mkstemp(dir=str(cert_path.parent), prefix='.tmp')
```

Comment:
Oh ew....

Reply:
Yeah, most of your comments were on code that was already there.